source/ceylon/test/assertions.ceylon

```diff
@@ -222,6 +223,20 @@ shared class ExceptionAssert(
     }
 }
 
+"Fails the test if the given value does not satisfy the provided ClassOrInterface"
+throws (`class AssertionError`, "When _actual_ does not satisfy _expected_.")
+shared void assertIs(
+        "The actual value to be checked."
+        Anything val,
+        "The class or interface to be satisfied."
+        ClassOrInterface<> expected,
+        "The message describing the problem."
+        String? message = null){
+    if (!expected.typeOf(val)){
+        throw AssertionError(message else "assertion failed: expected type not satisfied. expected <``expected``>");
```

Comment:
I think `type(val)` should be in the message since it's the unknown piece of info. Something like

    <``type(val)``> is not a subtype of <``expected``>